### DISCUSSION TITLE
Where are unsaved documments stored? Is there any "temp save" in case say my PC crashes whilst writing a text document?

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [0x7c13/Notepads#1038] [Q&A]
Where are unsaved documments stored? Is there any "temp save" in case say my PC crashes whilst writing a text document?

**Comment by 0x7c13** [comment]
C:\Users\<your username>\AppData\Local\Packages\19282JackieLiu.Notepads-Beta_echhpq9pdbte8\LocalState\BackupFiles

**Comment by Betaum1303** [comment]
It happens to me. My computer had problemas and When i started Notepads it was empty. All my work is gone. Is there any possibility of recover it. The mencioned files above are with a few kb

**Reply by 0x7c13** [reply]
Check above path to see if there are any files there. You can open them using notepad or other editors just to be save.

**Reply by Betaum1303** [reply]
Olhei. O arquivo só tem alguns Kb. Como disse, deveria haver backup de alguns dias. Quem sabe, da semana. Meu app deu problema ontem, quando abri ele estava vazio. Fui olhar esse arquivo na pasta citada e não havia nada lá. Meses de anotações se foram.

**Comment by Betaum1303** [comment]
Abrimos tantas abas não salvas no notepads que ele deveria guardar mais backups. Meu problema ocorreu ontem e eu já não tenho mais nada das minhas anotações feitas ao longo de mais de 6 meses.
